client/lib/products-values/index.js

```diff
@@ -2,11 +2,16 @@
  * External dependencies
  */
 import { assign, difference, get, includes, isEmpty, pick } from 'lodash';
+import moment from 'moment';
```

Comment:
`moment` is large, and in Calypso we're trying to ditch any direct usages like this in favor of the HoC and hook that we have. See below for a suggestion on how to approach that.